src/averageAges.js

```diff
@@ -37,7 +45,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => person.sex === 'f');
+  const allMothers = people.map(person => person.mother);
```

Comment:
```suggestion
  const motherNames = people.map(person => person.mother);
```

Reply:
Also, you could use `Set` for it. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set 
Set accepts an array, you will get a more efficient algorithm because checking the value that exists inside the set is efficient than using `includes`  in array